go.mod

```diff
@@ -1,86 +1,96 @@
 module github.com/DataDog/datadog-operator
 
-go 1.22
+go 1.22.0
 
 toolchain go1.22.7
 
 require (
 	github.com/DataDog/datadog-api-client-go/v2 v2.27.0
-	github.com/DataDog/extendeddaemonset v0.10.0-rc.4
+	github.com/DataDog/extendeddaemonset v0.9.0-rc.3.0.20241010101357-2f0cfa774277
```

Comment:
This is the commit of DataDog/extendeddaemonset#192.

To be updated as soon as this PR is merged.